core/src/main/resources/Samples/Input_Json_Specification.json

```diff
@@ -542,7 +548,7 @@
     "comment_product": "The product information for which this is being done",
     "product": "my_product",
     "comment_ec2instanceprofile": "The role with whioch the emr cluster(ec2-instances) has to be spun up so that they can access vault.",
```

Comment:
```suggestion
    "comment_ec2instanceprofile": "For DataPulls running on EMR, the AWS Role with which the EC2 nodes of the EMR cluster have to be spun up so that they can access Vault.",
```